types/types.go

```diff
@@ -332,6 +333,12 @@ type SystemContext struct {
 	DockerDisableV1Ping bool
 	// Directory to use for OSTree temporary files
 	OSTreeTmpDirPath string
+	// The hostname or IP to the docker server. If not set (aka ""), client.DefaultDockerHost is assumed.
```

Comment:
Please create a separate `=== docker/daemon.Transport overrides ===` section, and name the new members `DockerDaemon` to make them separate from the docker/distribution registry client.

Reply:
Got it now. This is a quite subtle difference.